php-practice/section5.php

```diff
@@ -320,3 +338,25 @@ public function send() {
  *   'cost' => 3.99
  * ];
  */
+
+class ShoppingCart {
+  private $item;
+  private $shoppingCart;
+
+  $item = [
+    'name' => 'Pringles',
+    'cost' => 3.99
+  ];
+
+  public function addItem() {
+    $shoppingCart =  [];
+    $shoppingCart[] = array_push($item);
```

Comment:
Also, I see that you're doing `$shoppingCart[]` **and** `array_push` in the same command.

That's not quite what you want -- they both do the same thing. You need to pick one or the other.

If you want to push `5` onto `$this->shoppingCart`, you can do one of the following:

```PHP
array_push($this->shoppingCart, 5);
$this->shoppingCart[] = 5; // Just shorthand for the line above
```

If you look at the [documentation for array_push](http://php.net/manual/en/function.array-push.php), you'll see that it `Returns the new number of elements in the array.`